### PR TITLE
Add image, image tag and registry endpoints

### PR DESCRIPTION
**Pull request by pchico83**

### CODE REVIEW

File: source/includes/image.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    "repository": "test-private-repo",
+    "type": "GITHUB"
+  },
+  "categories": [
+    {
+      "name": "test"
+    }
+  ],
+  "description": "Test image",
+  "docker_registry": "/api/v1/registry/registry.local/",
```
Comment (pchico83) [inline source/includes/image.md]: should we rename this to `registry`?

File: source/includes/image.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+  "image_url": "",
+  "in_use": false,
+  "is_private_image": true,
+  "name": "registry.local/user1/image1",
+  "public_url": "",
+  "resource_uri": "/api/v1/image/registry.local/user1/image1/",
+  "star_count": 0,
+  "starred": false,
```
Comment (pchico83) [inline source/includes/image.md]: remove this?

Comment (fermayo) [inline source/includes/image.md]: rename it to `jumpstart`

File: source/includes/image.md (hunk 1 above)
Comment (fermayo) [inline source/includes/image.md]: yes, why not

**Pull request closed without merging**